scripts/xtensa-build-all.sh

```diff
@@ -15,7 +15,7 @@ DEFAULT_PLATFORMS=(
 
 # Work in progress can be added to this "staging area" without breaking
 # the -a option for everyone.
-SUPPORTED_PLATFORMS=( "${DEFAULT_PLATFORMS[@]}" )
+SUPPORTED_PLATFORMS=( "${DEFAULT_PLATFORMS[@]}" acp_6_3 )
```

Comment:
I meant `SUPPORTED_PLATFORMS+=( mt8188 acp_6_3 )` of course :-(

Reply:
Or even better:

```shell
# Waiting for container work in progress
SUPPORTED_PLATFORMS+=( mt8188 )

# No plan to support in container (or replace with the appropriate comment)
SUPPORTED_PLATFORMS+=( acp_6_3 )
```

Reply:
I have made the changes accordingly.

thanks @marc-hb